src/database/standalone-sqlite.ts

```diff
@@ -1100,22 +1100,37 @@ export class StandaloneSqliteDatabaseWorker {
       });
     }
 
+    // Calculate resolved offset values for reuse
+    const rootParentOffset = dataRow?.root_parent_offset ?? coreRow?.root_parent_offset;
+    const dataOffset = dataRow?.data_offset ?? coreRow?.data_offset;
```

Comment:
This looks questionable. Core rows won't have those fields since they're specific to bundles.

Reply:
Addressed in f15713f96ff4a315cafaddfa1e145661bbdd1289. The code was fine, the name `coreRow` was misleading. It contains results for both transactions and data items.